content/documentation/device-examples/components/Camera.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Camera
+weight: 210
+tags: ["Camera"]
+---
+### Description
+The [Camera](https://github.com/Pi4J/pi4j-example-components/tree/Dev-Arcade/src/main/java/com/pi4j/components/components) (src/main/java/com/pi4j/components/components) is a template class, that you can use in your own Java-project.
+Currently, the code is only tested with a [Raspberry-Camera](https://www.raspberrypi.com/documentation/accessories/camera.html#introducing-the-raspberry-pi-cameras) and the [crowpi-image](/getting-started/crowpi/crowpi-os/).
+You can take pictures or videos, with or without a preview.
+
+{{% notice note %}}
+The camera class is using the bash-commands "libcamera-still" and "libcamera-vid". To use them, set up the raspberry with the following [Introduction](https://www.raspberrypi.com/documentation/accessories/camera.html#getting-started)
+{{% /notice %}}
+
+### Layout
+{{< gallery >}}
+{{< figure link="/assets/documentation/device-examples/pictures/Raspberry-Camera.png" caption="Raspberry Camera" caption-position="center" caption-effect="fade" >}}{{< /gallery >}}
```

Comment:
provide additional picture of the camera, the type designation should be clearly visible (maybe front and back)